docs/topics/mta-rn-known-issues-7-0-0.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-0_{context}"]
+= Known issues
+
+MTA version 7.0.0 has the following issues.
+
+.Lack of Unique IDs and Aria-Labels on Sections/Categories, Hindering Automation
+
+There is a lack of unique IDs applied to sections and categories, which hinders migration automation. link:https://issues.redhat.com/browse/MTA-1203[MTA-1203]
+
+.Support disconnected installations
+
+MTA does not support disconnected installations. link:https://issues.redhat.com/browse/MTA-17[MTA-17]
+
+.Cannot remove the Jira connection to a Wave that is later removed 
+
+It is not possible to remove a Jira connection to a Wave, after that Wave is later removed. link:https://issues.redhat.com/browse/MTA-1014[MTA-1014]
```

Comment:
`after` ... `later` is a tautology. it is either later or after

Reply:
Unless "Wave" is being used as a proper noun, it should not be capitalized in either the heading (see above) or the text. Please double-check whether "wave" is developer shorthand for "migration wave." If it is, write out "migration wave."

Reply:
It is a proper noun